### PR TITLE
Add responsive and fix styles

### PR DESCRIPTION
**Pull request by smyzin**
- fixed styles
- added responsive
- checked on apple mail macOS/iOS (fine), gmail web (no grid & shadow [cause support]), gmail iOS (cut off styles)

### CODE REVIEW

File: src/components/header.html [hunk 1]
```diff
@@ -1,164 +1,53 @@
-<tr>
-  <td class="p-16">
-    <div class="flex flex-row items-center space-x-8">
+<table class="w-full mt-32 sm:mt-0">
+  <tr class="inline">
+    <td class="items-center align-middle py-16 pl-16">
       <a href="https://sealcred.xyz" class="no-underline">
-        <svg
+        <img
```
Comment (ohpaul) [inline src/components/header.html:5]: Is using the img the reason why it doesn't show up on the email?

Comment (ohpaul) [reply src/components/header.html:5]: Instead of the svg

Comment (T-Damer) [reply src/components/header.html:5]: SVGs doesn't work in most email clients

Comment (T-Damer) [reply src/components/header.html:5]: https://stackoverflow.com/questions/37753911/how-can-i-embed-svg-into-html-in-an-email-so-that-its-visible-in-most-all-emai

**Review by T-Damer — APPROVED** [review]
lgtm

**Pull request merged**